server/portal/apps/auth/views.py

```diff
@@ -54,6 +58,26 @@ def agave_oauth(request):
     return HttpResponseRedirect(authorization_url)
 
 
+def launch_setup_checks(user):
+    """Perform any onboarding checks that may spawn celery tasks
+    """
+
+    # Check onboarding settings
+    new_user_setup_check(user)
+    if not user.profile.setup_complete:
```

Comment:
what's our way of adding new profile requirements in the future?

Reply:
@rstijerina , @jchuahtacc , going to merge this into our feature branch. let's follow up discussing there or in slack about this.